tensorboard/plugins/projector/vz_projector/vz-projector-bookmark-panel.ts

```diff
@@ -197,9 +197,12 @@ class BookmarkPanel extends LegacyElementMixin(PolymerElement) {
    */
   private getParentDataIndex(evt: Event) {
     for (let i = 0; i < (evt as any).path.length; i++) {
-      let dataIndex = (evt as any).path[i].getAttribute('data-index');
-      if (dataIndex != null) {
-        return +dataIndex;
+      let elem = (evt as any).path[i];
```

Comment:
optional + minor: ideally these `let`s would be `const`s, since they don't get updated.

Since it was there before, also fine with the PR as is.